test/widget_test.dart

```diff
@@ -5,26 +5,23 @@
 // gestures. You can also use WidgetTester to find child widgets in the widget
 // tree, read text, and verify that the values of widget properties are correct.
 
-import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
-import 'package:corona_trace/main.dart';
-
 void main() {
   testWidgets('Counter increments smoke test', (WidgetTester tester) async {
-    // Build our app and trigger a frame.
-    await tester.pumpWidget(MyApp());
-
-    // Verify that our counter starts at 0.
-    expect(find.text('0'), findsOneWidget);
-    expect(find.text('1'), findsNothing);
-
-    // Tap the '+' icon and trigger a frame.
-    await tester.tap(find.byIcon(Icons.add));
-    await tester.pump();
-
-    // Verify that our counter has incremented.
-    expect(find.text('0'), findsNothing);
-    expect(find.text('1'), findsOneWidget);
+//    // Build our app and trigger a frame.
```

Comment:
Curious about these being commented out?

Reply:
the test does not work anyway and it marks the files in my IDE as red

Reply:
Okay. Can we add a TODO or a FIXME so that someone can pick it up later as required.

Reply:
Sure why not. @SAGARSURI and I would have added tests after a few more housekeeping PRs anyway for the whole app